docs/user/algo_sac.md

```diff
@@ -0,0 +1,62 @@
+# Soft Actor-Critic
+
+```eval_rst
+.. list-table::
+   :header-rows: 0
+   :stub-columns: 1
+   :widths: auto
+
+   * - **Paper**
+     - Soft Actor-Critic Algorithms and Applications :cite:`haarnoja2018soft`
+   * - **Framework(s)**
+     - .. figure:: ./images/pytorch.png
+        :scale: 10%
+
+        PyTorch
+   * - **API Reference**
+     - `garage.torch.algos.BC <../_autoapi/garage/torch/algos/index.html#garage.torch.algos.SAC>`_
+   * - **Code**
+     - `garage/torch/algos/bc.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/torch/algos/sac.py>`_
+   * - **Examples**
+     - `examples <algo_sac.html#examples>`_
+```
+
+Soft Actor-Critic (SAC) is an algorithm which optimizes a stochastic policy in
+an off-policy way, forming a bridge between stochastic policy optimization and
+DDPG-style approaches. A central feature of SAC is entropy regularization. The
+policy is trained to maximize a trade-off between expected return and entropy,
+a measure of randomness in the policy. This has a close connection to the
+exploration-exploitation trade-off: increasing entropy results in more
+exploration, which can accelerate learning later on. It can also prevent the
+policy from prematurely converging to a bad local optimum.
+
+## Default Parameters
+
+```python
+gradient_steps_per_itr=1000
+max_episode_length_eval=1000
```

Comment:
is this really the default? I thought that the default was `None`

Reply:
Sorry. I made a mistake here